EDDI/EDDI.cs

```diff
@@ -238,11 +238,11 @@ void ActionUpdateSquadronSystem()
 
         public bool ShouldUseTestEndpoints()
         {
-#if DEBUG
-            return true;
+#if RELEASE
```

Comment:
Bzzzt. There is no RELEASE symbol!

Reply:
I am fairly reminded that there is no RELEASE preprocessor symbol (unless we define one, which we don't want to do)